cypress/support/expect-playing-audio.ts

```diff
@@ -10,4 +10,14 @@ const expectPlayingAudio = () => {
   });
 };
 
-export default expectPlayingAudio;
+export const expectNoPlayingAudio = () => {
```

Comment:
Instead of duplicating the entire function and changing just one word, couldn't the common functionality be factored out to another function?

Reply:
I tried playing with it but it just didn't work well in any way, so I decided to just let it go. Suggestions welcome